src/commonMain/kotlin/au/id/micolous/metrodroid/transit/serialonly/HoloTransitData.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * HoloTransitData.kt
+ *
+ * Copyright 2015-2016 Michael Farrell <micolous+git@gmail.com>
+ * Copyright 2018 Google Inc.
+ *
+ * Authors: Vladimir Serbinenko, Michael Farrell, Trevor Nielsen
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package au.id.micolous.metrodroid.transit.serialonly
+
+import au.id.micolous.metrodroid.card.CardType
+import au.id.micolous.metrodroid.card.desfire.DesfireApplication
+import au.id.micolous.metrodroid.card.desfire.DesfireCard
+import au.id.micolous.metrodroid.card.desfire.DesfireCardTransitFactory
+import au.id.micolous.metrodroid.multi.Parcelize
+import au.id.micolous.metrodroid.multi.R
+import au.id.micolous.metrodroid.time.Epoch
+import au.id.micolous.metrodroid.time.MetroTimeZone
+import au.id.micolous.metrodroid.time.TimestampFormatter
+import au.id.micolous.metrodroid.transit.CardInfo
+import au.id.micolous.metrodroid.transit.TransitIdentity
+import au.id.micolous.metrodroid.transit.TransitRegion
+import au.id.micolous.metrodroid.ui.ListItem
+
+/**
+ * Transit data type for HOLO card.
+ *
+ *
+ * This is a very limited implementation of reading HOLO, because only
+ * little data is stored on the card
+ *
+ *
+ * Documentation of format: https://github.com/metrodroid/metrodroid/wiki/HOLO
+ */
+@Parcelize
+data class HoloTransitData(private val mSerial: Int?,
+                                private val mLastTransactionTimestamp: Int?, private val mManufacturingId: String?) : SerialOnlyTransitData() {
+
+    public override val extraInfo: List<ListItem>
+        get() = listOf(
+            ListItem(R.string.last_transaction, when (mLastTransactionTimestamp) {
+                0 -> "Never"
+                null -> "Never"
+                else -> TimestampFormatter.dateTimeFormat(parseTime(mLastTransactionTimestamp)).toString()

```

Comment:
Please keep FormattedString. You can e.g.:
ListItem(Localizer.localizeFormatted(R.string...), TimesampFormatter.dateTimeFormat